projection/bridge_activity/bridge_pending_activity/bridge_pending_activity.go

```diff
@@ -41,17 +40,44 @@ type Config struct {
 	StartingHeight        int64  `mapstructure:"starting_height"`
 }
 
+const (
+	MIGRATION_DIRECOTRY = "projection/bridge_activity/bridge_pending_activity/migrations"
+)
+
 func NewBridgePendingActivity(
 	logger applogger.Logger,
 	rdbConn rdb.Conn,
 	config *appprojection.Config,
 ) *BridgePendingActivity {
+
+	// Here we customize the migrationSourceURL, as it is not following the same path convention
+	// in other projections. (extra `bridge_activity` folder in the path)
+	migrationSourceURL := github_mh.GenerateSourceURL(
+		github_mh.MIGRATION_GITHUB_URL_FORMAT,
+		config.GithubAPIUser,
+		config.GithubAPIToken,
+		MIGRATION_DIRECOTRY,
+		config.MigrationRepoRef,
+	)
+
 	return &BridgePendingActivity{
 		rdbprojectionbase.NewRDbBaseWithOptions(
-			rdbConn.ToHandle(), "BridgePendingActivity", rdbprojectionbase.Options{
+			rdbConn.ToHandle(),
+			"BridgePendingActivity",
+
+			rdbprojectionbase.Options{
 				MaybeTable:     nil,
 				MaybeConfigPtr: &Config{},
-			}),
+				MaybeMigrationHelper: github_mh.NewGithubMigrationHelper(
+					github_mh.Config{
+						Config:     *config,
+						ConnString: rdbConn.(*pg.PgxConn).ConnString(),
+					},
+					migrationSourceURL,
+					"",
+				),
+			},
+		),
```

Comment:
Can we pass the migration helper as an argument too? That way user can decide to use non-GitHub way to instantiate this projection.

Reply:
Oh, I also introduced a `FilesystemMigrationHelper`. The MaybeMigrationHelper's type is `MigrationHelper` interface.

User could switch another type of migration helper.

Reply:
I think we can use migration helper as an projection object argument and use it in projection.OnInit().

Reply:
oh, I get it now. I will put the `XXXMigrationHelper` initialization logic on the caller side. Will change it accordingly!